ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -47,6 +53,9 @@ class FaultInjector
   ros::Subscriber m_joint_state_sub;
   ros::Publisher m_joint_state_pub;
 
+  ros::Publisher fault_status_pub;
+  ros::Publisher arm_fault_status_pub;
```

Comment:
Prefix these with "m_" like the other data members -- it's one of our coding guidelines (on Confluence).